service/src/runner.rs

```diff
@@ -12,17 +11,13 @@ use tracing::info;
 pub async fn start(
     beta: bool,
     port: u16,
-    // only used on beta. must match port.
-    address: SocketAddr,
     runtime_executable: PathBuf,
     project_path: &Path,
 ) -> anyhow::Result<(process::Child, runtime::Client)> {
```

Comment:
Will this break anything? For example if someone upgrades deployer, but not runtime? :thinking:

Reply:
Only the args passed to beta runtime changes, though, so it should be fine..

Reply:
Yeah that line was added as part of adding the beta runtime flag